src/app/project/project-documents/upload/upload.component.html

```diff
@@ -28,6 +28,15 @@
           <div class="invalid-notice" *ngIf="myForm.get('projectPhase').invalid && myForm.get('projectPhase').touched">Project Phase is required</div>
         </div>
       </div>
+      <div class="form-row">
+        <div class="form-group col-md-4">
```

Comment:
Not sure how important it is for this app to be responsive. Just wondering how this looks on different screen sizes since only the md cols are defined here.

Reply:
Great point! For the "admin" (backend) app, it's mobile is pretty much never considered. We don't really expect the client to be editing projects there. Appreciate the feedback tho!